Twig/RollbarExtension.php

```diff
@@ -128,8 +128,24 @@ function isFromAllowedHosts(filename) {
         return false;
     }
 
+    function isStringError(payload) {
```

Comment:
`isStringError` sounds a bit strange, I would suggest to call it like `isLogMessage` or `isMessage`.